server/src/main/java/org/apache/iotdb/db/sync/externalpipe/ExtPipePluginManager.java

```diff
@@ -54,32 +55,22 @@ public class ExtPipePluginManager {
   private Map<String, ExtPipePlugin> extPipePluginMap = new HashMap<>();
 
   private ExecutorService monitorService = Executors.newFixedThreadPool(1);
```

Comment:
Use `IoTDBThreadPoolFactory` to create ExecutorService in ExtPipePluginManager and ExtPipePlugin

Reply:
Thank comments from @Cpaulyz .
I have just corrected it.